activation/validation.go

```diff
@@ -169,17 +168,13 @@ func (*Validator) VRFNonce(nodeId types.NodeID, commitmentAtxId types.ATXID, vrf
 	return nil
 }
 
-func (*Validator) InitialNIPostChallenge(challenge *types.NIPostChallenge, atxs atxProvider, goldenATXID types.ATXID, expectedPostIndices []byte) error {
+func (*Validator) InitialNIPostChallenge(challenge *types.NIPostChallenge, atxs atxProvider, goldenATXID types.ATXID) error {
 	if challenge.Sequence != 0 {
 		return fmt.Errorf("no prevATX declared, but sequence number not zero")
 	}
 
-	if challenge.InitialPostIndices == nil {
-		return fmt.Errorf("no prevATX declared, but initial Post indices is not included in challenge")
-	}
-
-	if !bytes.Equal(expectedPostIndices, challenge.InitialPostIndices) {
-		return fmt.Errorf("initial Post indices included in challenge does not equal to the initial Post indices included in the atx")
```

Comment:
What replaces this check? Can the initial post now use any indices?

Reply:
Nothing. It checked if indices in the `NipostChallenge.InitialPostIndices` match indices in `InnerActivationTx.InitialPost.Indices`. Now there's no repetition of indices. The initial POST proof **is still verified** in a different place.